pkg/stats/statsmgr.go

```diff
@@ -11,19 +15,41 @@ import (
 type StatsMgr struct {
 	StatsCh       chan base.Stats
 	DoneCh        chan bool
+	statsRW       sync.RWMutex
 	NumFailed     int64
 	NumReadFailed int64
+	totalLine     int64
 	totalCount    int64
 	totalBatches  int64
 	totalLatency  int64
 	totalReqTime  int64
 }
 
-func NewStatsMgr(numReadingFiles int) *StatsMgr {
+type StatsQuery struct {
```

Comment:
How about define `StatsMgr` as follows?
```go
type StatsMgr struct {
	StatsCh       chan base.Stats
	DoneCh        chan bool
	statsRW       sync.RWMutex
  stats         Stats
}

type Stats struct {
	NumFailed     int64
	NumReadFailed int64
	TotalLine     int64
	TotalCount    int64
	TotalBatches  int64
	TotalLatency  int64
	TotalReqTime  int64
}
```

Then, you can define a function `func (s *StatsMgr) Stats() Stats` replace the current `StatsQuery` function.